pyproject.toml

```diff
@@ -58,6 +58,7 @@ wrapt = [
 packaging = "^23.1"
 pipdeptree = "^2.13.0"
 reflex-hosting-cli = ">=0.1.2"
+charset-normalizer = "^3.3.2"
```

Comment:
just out of curiosity, was `chardet` considered for this? it's what i've mostly seen used in the past, but i see it hasn't had a release for quite some time.

Reply:
Not really, I didn't look into that. I saw some online answers using this, and this seems to be a dependency of some of current packages already.
But I will double check what people say about those two before merging.

Reply:
oh nice, charset-normalizer _already_ being in our installed payload makes this an easy decision

Reply:
Oh ok. Let's use this package for now. It's in a rather tiny area of our code